AdminServer/appscale/admin/constants.py

```diff
@@ -69,5 +70,19 @@ class Types(object):
 # The seconds to wait for redeploys.
 REDEPLOY_WAIT = 20
 
-# A list of projects that cannot be used or modified by users.
-RESERVED_PROJECTS = [DASHBOARD_APP_ID]
+# A list of projects that cannot be modified by users.
+IMMUTABLE_PROJECTS = [DASHBOARD_APP_ID]
+
+# The ZooKeeper node that prevents concurrent location assignments.
+VERSION_UPDATE_LOCK_NODE = '/appscale/version_update_lock'
```

Comment:
Isn't it a good idea to put version node pattern with named placeholders near this one?